### ISSUE TITLE
String parsing: strange behaviour following newline \n

### ISSUE THREAD

**Issue by jimblackler** [brimworks/json5.java#1]
If I parse the string

```json
"hello\nworld"
```

`visit` is called back with `val` set to 

```
hello
wwoorrlldd
```

Tested with version 2.0.0 (and on a main branch build today).

This can be seen with this short test:

```java
public class TestString {
  public static void main(String[] args) {
    new JSON5Parser().setVisitor(new JSON5Visitor() {
      @Override
      public void visit(String val, int line, long offset) {
        System.out.println(val);
      }
    }).parse("\"hello\\nworld\"", "");
  }
}
```

**Comment by brimworks** [comment]
Well, I have fix, but it seems that bintray is shutdown, so I guess I need to figure out a new way to publish the maven artifacts.

**Comment by brimworks** [comment]
Okay, it looks like I successfully published a 2.0.1 version, please let me know if that works.
